pontoon/checks/libraries/pontoon.py

```diff
@@ -32,9 +34,19 @@ def run_checks(entity, string):
     checks = defaultdict(list)
     resource_ext = entity.resource.format
     max_length = get_max_length(entity.comment)
+    html_parser = HTMLParser.HTMLParser()
+    string_length = len(
+        html_parser.unescape(
+            bleach.clean(
+                string,
+                strip=True,
+                tags=()
+            )
+        )
+    )
 
     # Prevent translations exceeding the given length limit
```

Comment:
This comment should indicate the entire max length check. So it should be moved above the following line:
```
max_length = get_max_length(entity.comment)
```